en/05_Contributing/06_Documentation.md

```diff
@@ -1,141 +1,250 @@
+* If you are adding documentation for functionality that has not yet been released, you should target the most recent minor release branch branch (e.g. `5`).
 
 [warning]
-If you submit a new feature or an API change, we strongly recommend and request that you include a PR to update the necessary documentation. This helps prevent our documentation from getting out of date.
+You should make your changes in the lowest major branch they apply to. For instance, if you fix a spelling issue that you found in the CMS 4 documentation, submit your fix to the `4.13` branch in Github and it'll be copied to the most recent major version of the documentation automatically. *Don't submit multiple pull requests for the same change*.
 [/warning]
 
-## Repositories
+## Editing online
 
-* End-user help: [silverstripe/silverstripe-userhelp-content](https://github.com/silverstripe/silverstripe-userhelp-content/)
-* Developer guides: [silverstripe/developer-docs](http://github.com/silverstripe/developer-docs/)
-* Website code for user and developer docs: [silverstripe/doc.silverstripe.org](https://github.com/silverstripe/doc.silverstripe.org)
-* Developer API documentation: [silverstripe/api.silverstripe.org](http://github.com/silverstripe/api.silverstripe.org)
+If you haven't contributed documentation in the past, you will need to fork the repository. If you're editing online, GitHub will guide you through that process before you can edit the documentation.
 
-## Source control
+### Simple changes {#editing-online-simple}
 
-Contributing documentation is the same process as providing any other patch (see [Contributing code](code)).
+The easiest way of editing any documentation is by clicking the "Edit on GitHub" button at the bottom of the page you want to edit. This works great if all of the changes you want to make are on a single page.
 
-If you are fixing incorrect or incomplete information for the current major version, you should create a PR that targets the branch for the latest stable release (e.g. `4.11`).
+Before you start making your changes, take note of what branch you are editing. You will need this when you create the pull request.
 
-If you are adding documentation for functionality that has not yet been released, you should target the most recent _major_ branch (e.g. `4`)
+![note the branch you're editing](../_images/contributing/docs-check-the-branch.png)
 
-## What to write
+After editing the documentation, click "Commit changes", then describe your changes in the "commit summary" and "extended description" fields below.
 
-See [what to write (jacobian.org)](http://jacobian.org/writing/great-documentation/what-to-write/) for an excellent introduction to the different types of documentation. Also see [producing OSS: "documentation"](http://producingoss.com/en/getting-started.html#documentation) for good rules of thumb 
-for documenting open source software.
```

Comment:
What to write? Write documentation. These links were just adding more (and frankly outdated) burden to anyone trying to contribute.